examples/deleteTokens.py

```diff
@@ -0,0 +1,43 @@
+'''
+Created on 17 March 2016
+
+@author: Natalie Danezi <anatoli.danezi@surfsara.nl>
```

Comment:
Remove date and author for consistency with the other scripts. Keep reference to servicedesk